src/EventListener/LoggerListener.php

```diff
@@ -14,13 +15,21 @@
  * Sets the username from the security context by listening on kernel.request
  *
  * @author Christophe Coevoet <stof@notk.org>
+ *
+ * @phpstan-template T of Loggable|object
```

Comment:
`Loggable|object` is the same than `object`

Reply:
used the same definition as in [LoggableListener](https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/src/Loggable/LoggableListener.php#L40), so what should I do here?

Reply:
Either leave it as is or just make it `T of object`.  `Gedmo\Loggable\Loggable` is an optional marker interface folks can use in their applications if they want to mark their models as loggable, but nothing requires implementing it and the only references to it in the library are template annotations.